internal/controller/operatorconfigmap_controller.go

```diff
@@ -882,6 +900,18 @@ func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscription() erro
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) reconcileNoobaaOperatorSubscription() error {
+	noobaaSubscription, err := c.getSubscriptionByPackageName("noobaa-operator")
+	if err != nil {
+		return err
+	}
+	noobaaSubscription.Spec.Channel = utils.GetNoobaaOperatorChannel()
+	if err := c.update(noobaaSubscription); err != nil {
+		return fmt.Errorf("failed to update subscription channel of 'noobaa-operator' to %v: %v", c.subscriptionChannel, err)
+	}
```

Comment:
This will cause an update on the resource even if the value never changes, which in turn will trigger the reconcile on every controller that watches this resource